penny_university/settings/base.py

```diff
@@ -208,3 +208,4 @@
 # Background Tasks
 CHAT_REMINDER_BEFORE_PENNY_CHAT_MINUTES = 75  # extra 15 minutes to make sure we remind them MORE than an hour in advance
 FOLLOWUP_REMINDER_AFTER_PENNY_CHAT_MINUTES = 30
+MAX_ATTEMPTS = 3
```

Comment:
MAX_ATTEMPTS for what? I'm not seeing this anywhere else in the code...

Reply:
I added this comment `stupidly, generically-named setting that controlled how many time a background_task is run` - why would they make sure a generic name for an environ variable?